src/screens/TimeCounterScreen.tsx

```diff
@@ -43,9 +43,9 @@ export const TimeCounterScreen = (props: Props) => {
     return true;
   };
 
-  const onFinish = () => {
-    if (onCountFinish) {
-      onCountFinish();
+  const onTryAgainPress = () => {
+    if (onTryAgain) {
```

Comment:
`onTryAgain` is marked as it has to be in route.params, so this if seems to be useless